mobilecoind-json/src/data_types.rs

```diff
@@ -1547,3 +1547,16 @@ mod test {
         assert_eq!(proto_proposal, proto2);
     }
 }
+
+#[derive(Serialize, Default, Debug)]
+pub struct JsonMobilecoindVersionResponse {
+    pub version: String,
+}
+
+impl From<&mc_mobilecoind_api::MobilecoindVersionResponse> for JsonMobilecoindVersionResponse {
+    fn from(src: &mc_mobilecoind_api::MobilecoindVersionResponse) -> Self {
+        Self {
+            version: src.get_version().to_string(),
```

Comment:
IIUC we can just reference the proto field
```suggestion
            version: src.version,
```

Reply:
Doesn't seem to work since version is a String, and the string type does implement the Copy trait ( see below for cargo build error message)

And I guess get_version() returns `&str` so I have to `to_string()` it ... is there something else I should try?
```
error[E0507]: cannot move out of `src.version` which is behind a shared reference
    --> mobilecoind-json/src/data_types.rs:1559:22
     |
1559 |             version: src.version,
     |                      ^^^^^^^^^^^ move occurs because `src.version` has type `std::string::String`, which does not implement the `Copy` trait
```

Reply:
Ah OK. You could write `src.version.clone()`, which is equivalent, so we might as well keep it as-is.